src/tests/ftest/pool/target_query.yaml

```diff
@@ -0,0 +1,89 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+    - server-E
+  test_clients:
+    - client-A
+timeout: 800
+setup:
+  start_agents_once: False
+  start_servers_once: False
```

Comment:
Why is this necessary? Is it not enough for each mux variant to just create a new pool?

Reply:
Yes its copy paste. let me remove it